examples/mp2660/default/main/main.c

```diff
@@ -0,0 +1,227 @@
+#include <inttypes.h>
```

Comment:
Please format source with `clang-format`. Repository contains the clang-format settings

Reply:
Done.